src/main/java/org/datadog/jmxfetch/reporter/JsonReporter.java

```diff
@@ -38,14 +40,25 @@ protected void sendMetricPoint(
         metrics.add(metric);
     }
 
+    private ServiceCheck.Status statusToServiceCheckStatus(String status) {
+        if (status == Status.STATUS_OK) {
+            return ServiceCheck.Status.OK;
+        } else if (status == Status.STATUS_WARNING) {
+            return ServiceCheck.Status.WARNING;
+        } else if (status == Status.STATUS_ERROR) {
+            return ServiceCheck.Status.CRITICAL;
+        }
+        return ServiceCheck.Status.UNKNOWN;
+    }
+
     /** Use the service check callback to display the JSON. */
     public void doSendServiceCheck(String checkName, String status, String message, String[] tags) {
         log.debug("Displaying JSON output");
         Map<String, Object> sc = new HashMap<String, Object>();
-        sc.put("check", checkName);
```

Comment:
Should we move the creation of the name to a level above in `Reporter. sendServiceCheck`?

Example: https://github.com/DataDog/jmxfetch/pull/281/files#diff-fbc4f513fdb38556ab6696eb273a6da2R158-R164